### ISSUE TITLE
Tensorboard

### ISSUE THREAD

**Issue by LoekvanKooten** [ModernMT/MMT#394]
@davidecaroselli A really noobish question, but if I go to Tensorboard during training and click on graph, all I see is numbers. There should be a setting somewhere that shows you the actual words that are being trained on. How do I get those? Tensorboard is pointing to the tmp/neural_model directory.

**Comment by LoekvanKooten** [comment]
And how would I display a graph of the loss value, so that it's easier to find a good moment to interrupt the training?

**Comment by davidecaroselli** [comment]
Hi @LoekvanKooten 

> There should be a setting somewhere that shows you the actual words that are being trained on. How do I get those? 

I'm not aware of this feature of Tensorboard unfortunately, I suggest to post this question on the Tensorflow/Tensor2Tensor repo.

> And how would I display a graph of the loss value

Open Tensorboard and then, on the top-right of the screen, click "SCALARS" tab. On the main window now you should be able to see "global_step" and "learning_rate" graphs. Right after those, there is a "loss" label: click on it to open the "loss" graph.

**Comment by LoekvanKooten** [comment]
No, this does not work: "No scalar data is found". (This is during the actual training phase.)

**Comment by nicolabertoldi** [comment]
In order to correctly visualize plots of the training status

run tensorboard as follows
tensorboard --logdir=./runtime/default/tmp/training/neural_model

and access via browser this page
http://URL:6006/

where URL is the IP of the machine where tensorboard is running.
